stata_kernel/__init__.py

```diff
@@ -1,11 +1,3 @@
 """An example Jupyter kernel"""
 
-import traceback
-
 __version__ = '1.12.2'
-
-try:
-    from .kernel import StataKernel
-except:
-    print('Cannot import kernel')
-    traceback.print_exc()
```

Comment:
I'm curious if there will be a regression because of these changes. We've had some changes here lately https://github.com/kylebarron/stata_kernel/pull/363, https://github.com/kylebarron/stata_kernel/pull/377

Reply:
The StateKernel import is moved to `stata_kernel/__main__.py` instead of `stata_kernel/__init__.py`, but unsure whether the try/except block is still needed then. It seems harmless to have it so I added it back.

Reply:
Ouch! I am sorry that for some reason this conversation did not make it to my inbox. The problem with having this here is that you now import `stata_kernel.kernel` even when you do not need to, which then imports dependencies that may not be present. In particular, you shouldn't necessarily need to important `stata_kernel.kernel` to do `stata_kernel.install`.